src/quo/components/profile/expanded_collectible/view.cljs

```diff
@@ -32,45 +34,55 @@
     label]])
 
 (defn view-internal
-  [{:keys [container-style square? status on-press counter image-src] :or {status :default}}]
-  (let [theme                       (quo.theme/use-theme-value)
-        [image-size set-image-size] (rn/use-state {})]
-    (rn/use-effect
-     (fn []
-       (promesa/let [[image-width image-height] (rn/image-get-size image-src)]
-         (set-image-size {:width        image-width
-                          :height       image-height
-                          :aspect-ratio (/ image-width image-height)})))
-     [image-src])
-    [rn/pressable
-     {:on-press            on-press
-      :accessibility-label :expanded-collectible
-      :style               (merge container-style style/container)}
-     (case status
-       :unsupported [fallback-view
-                     {:label   (i18n/label :t/unsupported-file)
-                      :counter counter
-                      :theme   theme}]
-       :cant-fetch  [fallback-view
-                     {:label   (i18n/label :t/cant-fetch-info)
-                      :counter counter
-                      :theme   theme}]
-       [rn/view
-        [rn/image
-         {:style  (style/image square? (:aspect-ratio image-size))
-          :source image-src}]
-        [counter-view counter]])]))
+  []
+  (let [image-error? (reagent/atom false)]
```

Comment:
@mmilad75, If you notice on line 42 this component is already using react-state. 
`[image-size set-image-size] (rn/use-state {})`
so we should definitely only be using react state in this component since the lifecycles of react and reagent might clash and you could run into some strange errors. 👽

Reply:
I replaced all of the `reagent/atom` with react states.